hadoop-ozone/ozonefs-lib-current/pom.xml

```diff
@@ -83,6 +63,78 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <artifactSet>
+                <excludes>
+                  <exclude>classworlds:classworlds</exclude>
```

Comment:
What is this classworlds:classworlds, is this mistakenly added from maven example?

Reply:
Do we need this?

Reply:
No, we don't. Thanks to point to it. Removed.